docs/contributing/contributing-releases/README.md

```diff
@@ -46,7 +46,7 @@ Follow the steps below to create an RC release.
 
    ```bash
    git checkout main
-   git checkout -b <USERNAME>/<BRANCHNAME>
+   git checkout -b release/<MAJOR.MINOR>
```

Comment:
The release branch gets created as part of the process so we don't want to update this to the release branch

Reply:
We may also want to reopen the PR under a different branch name. Since this is the name of the release branch that gets created as part of the release process, we may be adding unintentional commits to the release branch

Reply:
OK thank you - I will recreate the PR with a different branch name and delete this one.